tools/codemod/src/v4.0.0/full.ts

```diff
@@ -39,3 +41,140 @@ export default function transformer(file: FileInfo, api: API, options: Options)
 
   return src;
 }
+
+function renameValue(from: string, to: string): Plugin {
+  return {
+    postcssPlugin: `rename-value ${from} to ${to}`,
+    Declaration(decl) {
+      const parsed = valueParser(decl.value);
+      parsed.walk((node) => {
+        if (node.value === from)
+          node.value = to;
+        if (node.value === `#{${from}}`)
+          node.value = to;
+        if (node.value === `#{${from}`)
+          node.value = `#{${to}`;
+        if (node.value === `${from}}`)
+          node.value = `${to}}`;
+      });
+      decl.value = parsed.toString();
+    }
+  };
+}
+
+function removeImport(importParam: string): Plugin {
+  return {
+    postcssPlugin: `remove-import ${importParam}`,
+    AtRule(rule) {
+      if (rule.name !== "import")
+        return;
+      const noQuotes = rule.params.slice(1, -1);
+      if (noQuotes !== importParam)
+        return;
+      rule.remove();
+    }
+  };
+}
+
+function toModulePaths(getPath: (module: string) => string) {
+  return ["cjs", "esm"].map((module) => getPath(module));
+}
+
+const importPathsToRemove = [
+  ...toModulePaths((module) => `~@itwin/core-react/lib/${module}/core-react/button/blue-large`),
```

Comment:
I think the `~` may not always be there depending on the configuration (it used to be always there, but I think it is no longer necessary, so some may not have it)

Reply:
Fixed.